### PR TITLE
added technologies used to readme

### PR DESCRIPTION
**Pull request by kyltec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Main Street Brewing Rewards Program 🍺
 
+## Technology Used
+
+- React
+- React Native
+- Xcode
+- GraphQL
+  - GraphCool
```
Comment (siddharthparmar7) [inline README.md]: GraphCool is not part of GraphQL. 
React Native includes React, no need to mention it.
You are not using XCode for any dev purposes.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes and add more details to your pull request (i.e, assign yourself, what sprint? etc).

File: README.md (hunk 1 above)
Comment (kyltec) [reply README.md]: ok il make changes

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Main Street Brewing Rewards Program 🍺
 
+## Technology Used
+
+- React
+- GraphQL
+- GraphCool
```
Comment (siddharthparmar7) [inline README.md:6]: Just say you are using GraphCool. No need to mention GraphQL.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**